tests/backup/backup_test.go

```diff
@@ -1828,186 +1828,184 @@ func killStork() {
+		  Inst().V.String(), storageNodes[0].Name, backupName), func() {
+		  // Just restart storage driver on one of the node where volume backup is in progress
+		  Inst().V.RestartDriver(storageNodes[1], nil)
 		})
+
+		Step("Check if backup is successful when the PX restart happened", func() {
+			var bkpUid string
+			backupDriver := Inst().Backup
+			ctx, err := backup.GetAdminCtxFromSecret()
+			for _, namespace := range bkpNamespaces {
+				backupName := fmt.Sprintf("%s-%s-%s", BackupNamePrefix, namespace, backupLocationName)
+				backupSuccessCheck := func() (interface{}, bool, error) {
+					bkpUid, err = backupDriver.GetBackupUID(ctx, backupName, orgID)
+					log.FailOnError(err, "Failed while trying to get backup UID for - %s", backupName)
+					backupInspectRequest := &api.BackupInspectRequest{
+						Name:  backupName,
+						Uid:   bkpUid,
+						OrgId: orgID,
+					}
+					resp, err := backupDriver.InspectBackup(ctx, backupInspectRequest)
+					log.FailOnError(err, "Inspecting the backup taken with request:\n%v", backupInspectRequest)
+					actual := resp.GetBackup().GetStatus().Status
+					expected := api.BackupInfo_StatusInfo_Success
+					if actual != expected {
+						return "", true, fmt.Errorf("backup status expected was [%s] but got [%s]", expected, actual)
+					}
+					return "", false, nil
+			}
+			task.DoRetryWithTimeout(backupSuccessCheck, 10*time.Minute, 30*time.Second)
+			bkpUid, err = backupDriver.GetBackupUID(ctx, backupName, orgID)
+			log.FailOnError(err, "Failed while trying to get backup UID for - %s", backupName)
+			backupInspectRequest := &api.BackupInspectRequest{
+				Name:  backupName,
+				Uid:   bkpUid,
+				OrgId: orgID,
+			}
+			resp, err := backupDriver.InspectBackup(ctx, backupInspectRequest)
+			log.FailOnError(err, "Inspecting the backup taken with request:\n%v", backupInspectRequest)
+			dash.VerifyFatal(resp.GetBackup().GetStatus().Status, api.BackupInfo_StatusInfo_Success, "Inspecting the backup success for - "+resp.GetBackup().GetName())
+		}
 	})
-	AfterEach(func() {
-		EndTorpedoTest()
+})
+JustAfterEach(func() {
+	defer EndTorpedoTest()
+	log.InfoD("Deleting the deployed apps after the testcase")
+	for i := 0; i < len(contexts); i++ {
+		opts := make(map[string]bool)
+		opts[SkipClusterScopedObjects] = true
+		taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+		err := Inst().S.Destroy(contexts[i], opts)
+		dash.VerifySafely(err, nil, fmt.Sprintf("Verify destroying app %s, Err: %v", taskName, err))
+	}
+
+	log.InfoD("Deleting backup location and cloud setting")
+	DeleteBackupLocation(backupLocation, backupLocationUID, orgID)
+	// Need sleep as it takes some time for
+	time.Sleep(time.Minute * 2)
+	for CloudCredUID, CredName := range CloudCredUIDMap {
+			DeleteCloudCredential(CredName, orgID, CloudCredUID)
+	}
+	DeleteCluster(destinationClusterName, orgID,ctx)
+	DeleteCluster(SourceClusterName, orgID, ctx)
```

Comment:
Here we can use the function ' DeleteCloudAccounts' instead of the above 4 lines

Reply:
Done